examples/combined_ODNP.py

```diff
@@ -142,110 +127,66 @@
-DNP_data.name("enhancement")
-DNP_data.chunk("t", ["ph1", "t2"], [4, -1])
+DNP_data.set_prop("postproc_type", "spincore_ODNP_v3")
+DNP_data.set_prop("acq_params", parser_dict.asdict())
+DNP_data.name(parser_dict['type'])
+DNP_data.chunk("t", ["ph1", "t2"], [len(Ep_ph1_cyc), -1])
 DNP_data.setaxis("ph1", Ep_ph1_cyc / 4)
+DNP_data.setaxis('nScans',r_[0:parser_dict['nScans']])
 logger.info("SAVING FILE... %s" % myfilename)
 DNP_data.hdf5_write(myfilename)
 logger.info("FILE SAVED")
 logger.debug(strm("Name of saved enhancement data", DNP_data.name()))
-logger.debug("shape of saved enhancement data", psp.ndshape(DNP_data))
+logger.debug("shape of saved enhancement data", ndshape(DNP_data))
 # }}}
 # {{{run IR
 with power_control() as p:
     retval_IR = p.dip_lock(
-        uw_dip_center_GHz - uw_dip_width_GHz / 2,
-        uw_dip_center_GHz + uw_dip_width_GHz / 2,
+        parser_dict['uw_dip_center_GHz'] - parser_dict['uw_dip_width_GHz'] / 2,
+        parser_dict['uw_dip_center_GHz'] + parser_dict['uw_dip_width_GHz'] / 2,
     )
     p.mw_off()
+    time.sleep(16)
     ini_time = time.time()  # needed b/c data object doesn't exist yet
     vd_data = run_IR(
         nPoints=nPoints,
-        nEchoes=nEchoes,
+        nEchoes=parser_dict['nEchoes'],
         vd_list_us=vd_list_us,
-        nScans=nScans,
-        adcOffset=adcOffset,
-        carrierFreq_MHz=carrierFreq_MHz,
-        p90_us=p90_us,
-        tau_us=tau_us,
-        repetition=FIR_rd,
-        output_name=output_name,
-        SW_kHz=SW_kHz,
+        nScans=parser_dict['nScans'],
+        adcOffset=parser_dict['adc_offset'],
+        carrierFreq_MHz=parser_dict['carrierFreq_MHz'],
+        p90_us=parser_dict['p90_us'],
+        tau_us=parser_dict['tau_us'],
+        repetition=parser_dict['FIR_rep'],
+        output_name=filename,
+        SW_kHz=parser_dict['SW_kHz'],
         ph1_cyc=IR_ph1_cyc,
         ph2_cyc=IR_ph2_cyc,
         ret_data=None,
     )
     vd_data.set_prop("start_time", ini_time)
     vd_data.set_prop("stop_time", time.time())
-    meter_power = -999
-    acq_params = {
-        j: eval(j)
-        for j in dir()
-        if j
-        in [
-            "adcOffset",
-            "carrierFreq_MHz",
-            "amplitude",
-            "nScans",
-            "nEchoes",
-            "p90_us",
-            "deadtime_us",
-            "repetition_us",
-            "SW_kHz",
-            "nPoints",
-            "deblank_us",
-            "tau_us",
-            "MWfreq",
-            "acq_time_ms",
-            "meter_power",
-        ]
-    }
-    vd_data.set_prop("acq_params", acq_params)
-    vd_data.set_prop("postproc_type", IR_postproc)
+    vd_data.set_prop("acq_params", parser_dict.asdict())
+    vd_data.set_prop("postproc_type", "spincore_IR_v1")
     vd_data.name("FIR_noPower")
-    vd_data.chunk("t", ["ph1", "ph2", "t2"], [len(IR_ph1_cyc), len(IR_ph2_cyc), -1])
+    vd_data.chunk("t", ["ph2", "ph1", "t2"], [len(IR_ph1_cyc), len(IR_ph2_cyc), -1])
```

Comment:
this was backwards! not sure if an error in ppg but we can worry about that later, for now this chunks appropriately and the resulting signal shows up in the right CT path